File: DotNet/Framework/AutomationFramework/Tests.UITests/DashboardPageTests/DashboardActiveMenuTest.cs
using NUnit.Framework;
using Pages;

namespace Tests.UITests.DashboardPageTests
{
    [TestFixture]
    public class DashboardActiveMenuTest : BaseTestFixtures
    {
        DashboardPage Dashboard_Page;

        [SetUp]
        public void SetUp()
        {
            Dashboard_Page = LoginUser();
        }

        [Test]
        public void MenuTest() 
        {
            Dashboard_Page.GetActiveMenuText(out string activeMenu);

            Assert.That(activeMenu, Is.EqualTo("Dashboard"));
        }
    }
}

File: Java/AutomationFramework/src/com/app/UITests/DashboardPageTests/DashboardActiveMenuTest.java
package com.app.UITests.DashboardPageTests;

import com.app.pages.DashboardPage;
import org.testng.Assert;
import org.testng.annotations.BeforeMethod;
import org.testng.annotations.Test;

public class DashboardActiveMenuTest extends BaseDashboardTest{
    DashboardPage Dashboard_Page;

    @BeforeMethod
    public void SetUp()
    {
        Dashboard_Page = LoginUser();
    }

    @Test
    public void MenuTest()
    {
        String activeMenu = Dashboard_Page.GetActiveMenuText();

        Assert.assertEquals(activeMenu, "Dashboard");
    }
}
